app/util/bitbucket/populate_db.sh

```diff
@@ -94,7 +94,27 @@ else
   echo "Postgres client is already installed"
 fi
 
-echo "Step2: Download DB dump"
+echo "Step2: Get DB Host"
+DB_HOST=$(sudo su -c "cat ${DB_CONFIG} | grep 'jdbc:postgresql' | cut -d'/' -f3 | cut -d':' -f1")
+if [[ -z ${DB_HOST} ]]; then
+  echo "DataBase URL was not found in ${DB_CONFIG}"
+  exit 1
+fi
+echo "DB_HOST=${DB_HOST}"
+
+echo "Step3: Write 'instance.url' property to file"
```

Comment:
Need to handle re-run case.
E.g. populate db step failed -> base_url exists and not empty, but the value in DB could be missing or incorrect.

Reply:
Add check if the file already exists and not empty